content/guides/embed/ui-elements/explorer.md

```diff
@@ -264,7 +271,162 @@ more, see [Dedicated Scopes for Box UI Elements][scopes].
+2. Apply an already created template to a Box folder. Make sure you enable the cascade policy. For detailed instructions, see [instructions on customizing and applying templates][apply-templates].
+
+### Display Metadata View
+
+To make things easier, you can use a [sample project][metadata-project] to launch metadata view.
+
+  1. Clone the metadata sample project.
+  2. Update the placeholders in [`App.js`][appjs] with actual values:
+
+      | Parameter          |  Description |
+      |--------------------|-----------------|
+      |`DEVELOPER_TOKEN`   |[Developer token][token] generated in the the Developer Console. |
+      |`ENTERPRISE_ID`     |Enterprise ID copied from the **General Settings** tab of your application.|
+      |`METADATA_TEMPLATE_NAME`  |Name of your already created metadata template.|
+      |`ROOTFOLDER_ID`   |  ID of Box folder to which you applied the metadata template. |
+
+The `defaultView`, `fieldsToShow`, and `metadataQuery` parameters are already defined in the sample project, as in the example below:
+
+```js
+function App() {
+	// Get the token from Developer Console (app's configuration tab)
+	const token = "<DEVELOPER_TOKEN>";
+
+	// Folder ID with a metadata template applied
+	// The metadataQuery will apply to this folder
+	const rootFolderID = "<ROOTFOLDER_ID>";
+
+	// Get ENTERPRISE_ID from Developer Console (app's general settings)
+	const EID = "<ENTERPRISE_ID>";
+
+	// Get templatekey from Admin Console (Content -> Metadata -> check url for ID)
+	const templateName = "<METADATA_TEMPLATE_NAME>";
+
+	// Define metadata source
+	// Example: enterprise_123456789.metadatatemplate
+	const metadataSource = `enterprise_${EID}.${templateName}`;
+
+	const metadataQuery = {
+		from: metadataSource,
+
+		// Filter items in the folder by existing metadata key
+		query: "key = :arg1",
+
+		// Display items with value
+		query_params: { arg1: "value" },
+
+		// Define the ancestor folder ID
+		ancestor_folder_id: 0,
+
+		// Define which other metadata fields you'd like to display
+		fields: [
+			`${metadataSource}.name`,
+			`${metadataSource}.last_contacted_at`,
+			`${metadataSource}.industry`,
+			`${metadataSource}.role`,
+		],
+	};
+
+	// The metadata fields/columns to view - must be valid field names from the metadata template
+	const fieldsToShow = [
+		// Determine if the user can edit the metadata directly from Content Explorer component
+		{ key: `${metadataSource}.name`, canEdit: false },
+
+		// Determine label alias on metadata column with displayName prop
+		{ key: `${metadataSource}.industry`, canEdit: false, displayName: "alias" },
+		{ key: `${metadataSource}.last_contacted_at`, canEdit: true },
+		{ key: `${metadataSource}.role`, canEdit: true },
+	];
+
+	// defaultView - a required prop to paint the metadata view.
+	// If not provided, you'll get regular folder view.
+	const defaultView = "metadata";
+
+	return (
+		<IntlProvider locale="en">
+			<div className="App">TODO</div>
```

Comment:
Instead of TODO you can paste this code
<div className="metadata-based-view">
              <ContentExplorer
                rootFolderId={rootFolderID}
                token={token}
                metadataQuery={metadataQuery}
                fieldsToShow={fieldsToShow}
                defaultView={defaultView}
              />
            </div>

Reply:
`<div className="metadata-based-view">
              <ContentExplorer
                rootFolderId={rootFolderID}
                token={token}
                metadataQuery={metadataQuery}
                fieldsToShow={fieldsToShow}
                defaultView={defaultView}
              />
            </div>`

Reply:
Changed 👍